book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  def initialize(title, author)
+    @title = title,
+             @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author, :rentals
+
+  def add_rental(person, date)
+    Rental.new(date, self, person)
+  end
+end
```

Comment:
- [ ] Please move `attr_accessor :title, :author, :rentals` straight bellow `class Book` because you want to tell Ruby create methods for you with `attr_accessor`

Reply:
Thanks @OLIPLICHE